pocs/core.py

```diff
@@ -478,7 +481,7 @@ def has_ac_power(self, stale=90):
 # Convenience Methods
 ##################################################################################################
 
-    def sleep(self, delay=2.5, with_status=True):
+    def sleep(self, delay=2.5, with_status=True, **kwargs):
```

Comment:
Does this **kwargs do anything?

Reply:
Yes, in the sense that it allows `sleep` to ignore other kwargs it doesn't care about. On line 593 I want to pass a series of kwargs to `wait_until_safe`, which in turn passes those to _both_ `is_safe` and `sleep`. If `is_safe` and `sleep` receive a kwarg they don't recognize they will throw an error. I could pull the sepcific parameters out of the `kwargs` in `wait_until_safe` and then pass them to `is_safe` and `sleep` respectively. But I'm lazy.

Really though this is just an effective way to say, "take the parameters you need and ignore all the others". I believe @jamessynge and I had debates about this a long time ago, as I used to include `*args, **kwargs` automatically on every method (although note that this is a different usage from subclassing).